ibis/backends/mysql/__init__.py

```diff
@@ -230,13 +230,15 @@ def _get_temp_view_definition(self, name: str, definition: str) -> str:
             expression=definition,
         )
 
+    # TODO(kszucs): should be moved to the base sqlglot backend
     def create_database(self, name: str, force: bool = False) -> None:
         sql = sge.Create(kind="DATABASE", exist=force, this=sg.to_identifier(name)).sql(
             self.name
         )
         with self.begin() as cur:
             cur.execute(sql)
 
+    # TODO(kszucs): should be moved to the base sqlglot backend
```

Comment:
I don't think so, most backends don't support it.

Reply:
Removed the note.